build-tools/packages/build-cli/src/commands/generate/typetests.ts

```diff
@@ -216,7 +217,20 @@ function getTypesPathWithFallback(
 export function getTypesPathFromPackage(
 	packageJson: PackageJson,
 	level: ApiLevel,
+	log: Logger,
 ): string | undefined {
+	if (packageJson.exports === undefined) {
+		log.verbose(`${packageJson.name}: No export map found.`);
+		// Use types/typings field only when the public API level is used and no exports field is found
+		if (level === ApiLevel.public) {
+			log.verbose(`${packageJson.name}: Using the types/typings field value.`);
+			return packageJson.types ?? packageJson.typings;
```

Comment:
Can we comment here or in TSDoc that this is meant to follow Typescript resolution (see https://www.typescriptlang.org/docs/handbook/modules/reference.html) and doesn't bother with "main" fallback because we don't have that?

Reply:
Done in latest.